src/context/WalletProvider/KeepKey/components/DownloadUpdater.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Icon, Link, ModalBody, ModalHeader, Text as CText } from '@chakra-ui/react'
+import { FaApple, FaLinux, FaWindows } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { Text } from 'components/Text'
+
+import { getPlatform } from '../helpers'
+
+export const KeepKeyDownloadUpdater = () => {
+  const platform = getPlatform()
+
+  const platformFilename =
+    platform === 'Mac OS'
+      ? 'KeepKey-Updater-2.1.4.dmg'
+      : platform === 'Windows'
+      ? 'KeepKey-Updater-Setup-2.1.4.exe'
+      : 'KeepKey-Updater-2.1.4.AppImage'
+
+  const platformIcon =
+    platform === 'Mac OS' ? FaApple : platform === 'Windows' ? FaWindows : FaLinux
+
+  const updaterUrl = `https://github.com/keepkey/keepkey-updater/releases/download/v2.1.4/${platformFilename}`
```

Comment:
Leaving this for reference since this is fine for now - no need to tackle it in this PR - the last release tag should be programmatic.

It can be fetched with `https://api.github.com/repos/keepkey/keepkey-updater/releases/latest`, although that's probably not something we want done at runtime and would rather belong in a simple script ran on build to avoid spamming GitHub API.

Reply:
Was concerned about calling the GH Api too since we'd need keys too to access this endpoint.